### PR TITLE
Add attribution fields

### PR DESCRIPTION
**Pull request by themoffatt**
Adding new attributions tables for 
* content_identifier
* medium_identifier
* campaign_identifier

### CODE REVIEW

File: lib/land/tracker.rb [hunk 1]
```diff
@@ -240,6 +243,7 @@ def create_visit
       visit.referer_id    = referer.try(:id)
       visit.user_agent_id = user_agent.id
       visit.ip_address    = remote_ip
+      visit.raw_query_string = request.query_string
```
Comment (spacechurro) [inline lib/land/tracker.rb:246]: @themoffatt How is this different than the normalized query string on `pageviews`?

Comment (themoffatt) [reply lib/land/tracker.rb:246]: This is the full query string before UTM params have been removed and normalized. Our marketing folks weren't trusting the normalization of the params and needed to be able to see the raw query string.

**Comment by tesstryan** [comment]
@themoffatt we still want to move forward with this PR? Happy to help/review if so!

**Comment by spacechurro** [comment]
@tesstryan I put cards in our backlog for pulling it into our apps.

**Comment by tesstryan** [comment]
Sounds good, should this be approved then?

**Comment by spacechurro** [comment]
Yep. My bad. Just approved and merged.

**Pull request merged**